snuba/datasets/entities/discover.py

```diff
@@ -88,8 +81,32 @@ def attempt_map(
             return None
 
 
-@dataclass(frozen=True)
+@dataclass
```

Comment:
You needed to remove `frozen` because you are assigning a variable in post_init, is that correct ?

Reply:
Yeah